tests/integ/scala/test_dataframe_join_suite.py

```diff
@@ -141,6 +141,9 @@ def test_join_with_multiple_conditions(session):
     assert res == []
 
 
+@pytest.mark.skip(
```

Comment:
I think we should run both sql simplifier and the original analyzer (probably you can build a separate merge gate for simplifier)